pkg/process/errors.go

```diff
@@ -20,6 +20,10 @@ const (
 	// windowsCommandNotFoundFragment is a fragment of the error output returned
 	// on Windows systems when a command cannot be found.
 	windowsCommandNotFoundFragment = "The system cannot find the path specified"
+	// windowsPowershellCommandNotFoundFragment is a fragment of the error output
+	// returned on Windows systems running Powershell when a command cannot be
+	// found.
+	windowsPowershellCommandNotFoundFragment = "is not recognized as the name of a cmdlet, function, script file, or operable program."
```

Comment:
This seems like it would break if you have a different OS language, but I guess mutagen was already doing similar error string checks previously and I can't really think of a better way to accomplish this. So it's probably fine for now